curt.py

```diff
@@ -1368,36 +1368,19 @@ def print_task_stats(tasks):
 					report_calls(task.irqs, irq_name, process.irqs, system.irqs)
 					print
 
-				# Adding per cpu per process info
-				# process.output_header()
-				print
-				############################################
-				#print 'Per CPU per process--START'
-				#for cpu in process.cpus:
-					#print "\t[%8s] %-20s %3u" % (pid, comm, cpu),
-					#process.cpus[cpu].output()
-					#print
-				#print 'Per CPU per process--END'
-				############################################
-
-				print "\t[     ALL] %-20s ALL" % (""),
-				process_cpus_sum.output()
-				process.output_migrations()
-				print
-				print
-
-
-		print 'Per CPU per process--START'
 		# Adding per cpu per process info
 		process.output_header()
 		print
-		############################################
 		for cpu in process.cpus:
-			print "\t[%8s] %-20s %3u" % (pid, comm, cpu),
+			print "\t[%8s] %-20s %3u" % ('ALL', comm, cpu),
```

Comment:
`comm` is left over from whatever the last task was. I'm not sure the perfect approach here, but I suggest one of
1. `'ALL'`
1. `'all'`
1. `''` (blank)
1. comparing the `comm` for all of the tasks in the process, and if they all agree, then use that, otherwise, one of the above.
1. change the column header to `pid` and output the process id in each row.